app/src/ui/changes/changes-list.tsx

```diff
@@ -286,22 +298,31 @@ export class ChangesList extends React.Component<
         ? false
         : null
 
-    const include =
-      rebaseConflictState !== null
-        ? file.status.kind !== AppFileStatusKind.Untracked
-        : includeAll
+    const include = isUncommittableSubmodule
+      ? false
+      : rebaseConflictState !== null
+      ? file.status.kind !== AppFileStatusKind.Untracked
+      : includeAll
+
+    const disableSelection =
+      isCommitting || rebaseConflictState !== null || isUncommittableSubmodule
 
-    const disableSelection = isCommitting || rebaseConflictState !== null
+    const checkboxTooltip = isUncommittableSubmodule
+      ? 'This submodule cannot be checked in because it contains changes have not been committed.'
```

Comment:
```suggestion
      ? 'This submodule cannot be checked in because it contains that changes have not been committed.'
```

Other thoughts are is the terminology "checked in" ideal - can't articulate what it is but it feels off to me - maybe it it is because in the UI we just use the word commit? Maybe this should be "This submodule change cannot be added to a commit in this repository, because..." or "This submodule change cannot be committed, because..."  

Similarly for next tooltip, "Only changes that have been committed within the submodule will be added to this repository" or "committed to this repository"

Reply:
Yeah I used `checked in` in an attempt to differentiate the actions in the main repo vs the actions in the submodule repo. I think "This submodule change cannot be added to a commit in this repository, because…" should be clear enough 🤔  Thanks for your suggestions!!